src/bundle/Controller/ContentViewController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * File containing the ContentViewController class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\HybridPlatformUiBundle\Controller;
+
+use eZ\Publish\API\Repository\Values\Content\Location;
+use eZ\Publish\Core\MVC\Symfony\Controller\Controller;
+use eZ\Publish\Core\MVC\Symfony\View\ContentView;
+
+class ContentViewController extends Controller
+{
+    public function detailsTabAction(ContentView $view)
+    {
+        $versionInfo = $view->getContent()->getVersionInfo();
+        $contentInfo = $versionInfo->getContentInfo();
+
+        $sectionService = $this->getRepository()->getSectionService();
+        $section = $sectionService->loadSection($contentInfo->sectionId);
+
+        $userService = $this->getRepository()->getUserService();
+        $creator = $userService->loadUser($contentInfo->ownerId);
+        $lastContributor = $userService->loadUser($versionInfo->creatorId);
+
+        $languageRepository = $this->getRepository()->getContentLanguageService();
+        $translations = [];
+        foreach ($versionInfo->languageCodes as $languageCode) {
+            $translations[] = $languageRepository->loadLanguage($languageCode);
+        }
+
+        $location = $view->getLocation();
+
+        $view->addParameters([
+            'section' => $section,
+            'contentInfo' => $contentInfo,
+            'versionInfo' => $versionInfo,
+            'creator' => $creator,
+            'lastContributor' => $lastContributor,
+            'translations' => $translations,
+            'ordering' => [
+                'sortFields' => $this->getSortFields($location->sortField),
+                'sortOrders' => $this->getSortOrders()
+            ]
+        ]);
+
+        return $view;
+    }
+
+    protected function getSortFields($currentSortField)
+    {
+        $defaultSortFields = [
```

Comment:
What are your thoughts on having these as constants or as protected variables of the class so that they could be extended in an easier fashion?

Would also make this function much smaller